tutorials/auto_inertia_calculation.md

```diff
@@ -0,0 +1,160 @@
+          <ambient>1.0 1.0 0.0 1.0</ambient>
+          <specular>1.0 1.0 0.0 1.0</specular>
+        </material>
+      </visual>
+    </link>
+  </model>
+```
+
+If you use the above snippet in an empty world and launch it with `gz-sim`, here's
+how it would look:
+
+![Cylinder](files/auto_inertia/cylinder_inertia_demo.gif)
+
+## Links with Multiple Collisions & the Effect of Density
+
+To understand the inertia calculation in links with multiple collisions and the
+effect of setting different density values, you can launch the `auto_inertia_pendulum.sdf`
+example world using:
+
+```bash
+gz sim auto_inertia_pendulum.sdf
+```
+
+After the gz-sim window opens up, you can right click on both the models and enable
+the centre of mass visualization by selecting the `View > Center of Mass` option from
+the menu. Once you play the simulation it should look this:
+
+![Pendulum](files/auto_inertia/auto_inertia_pendulum.gif)
+
+This example world has two structurally indentical models. The pendulum link of both
+the models contain 3 cylindrical collision geometries: One on the top which forms the
+joint, One in longer cylinder in middle and One at the end which forms the bob of
+cylinder. Even, though they are indentical, the center of mass for both are different
+as they use different density values for the different cylinder collisions. One one
+hand, the upper joint collision of the pendulum on the left has the highest density
+which causes the center of mass to shift closer to the axis while on the other hand,
+the bob collision of the pendulum on the right has the highest density which causes
+the center of mass to shift towards the end of the pendulum.
+This difference in mass distribution about the axis of rotation results in difference
+in the moment of inertia of the 2 setups and hence different angular velocities.
+
+## Mesh Inertia Calculation with Rolling Shapes Demo
+
+Let's try another example world, `auto_inertia_rolling_shapes.sdf`. This can be
+launched with `gz sim` using the following command:
+
+```bash
+gz sim auto_inertia_rolling_shapes.sdf
+```
+
+Once you launch and play the simulation, it should look something like this:
+
+![Rolling](files/auto_inertia/rolling_inertia_demo.gif)
+
+Here the right most shapes is a hollow cylinder (yellow). This model is load from
+fuel and is made using a collada mesh of a hollow cylinder. Apart from this, we can
+see there is a solid cylinder, a solid sphere and a solid capsule. All of these are
+made using the `<geometry>` tag and have automatic inertia calculations enabled.
+Here, the moments of inertia for the hollow cylinder (which is convex mesh shape) is
+calculated and from the simulation we can see that it reaches the bottom of the
+incline last. This is physically accurate as the mass distribution for the hollow
+cylinder is concetrated at a distance from the axis of rotation (which passes through
+the center of mass in this case).
+
+## Key Points Regarding Mesh Inertia Calculator

```

Comment:
```suggestion
## Key Points on Mesh Inertia Calculator

```

Reply:
Corrected in [2b8be4c](https://github.com/gazebosim/gz-sim/pull/2119/commits/2b8be4cfc423ee3b6ec37763861572a4362ec10b)